go/libraries/utils/argparser/results.go

```diff
@@ -135,7 +137,7 @@ func (res *ArgParseResults) DropValue(name string) *ArgParseResults {
 		}
 	}
 
-	return &ArgParseResults{newNamedArgs, res.Args, res.parser}
+	return &ArgParseResults{newNamedArgs, res.Args, res.parser, -1}
```

Comment:
Having -1 as a special number is not great. I'd make a descriptive constant, like "NO_DASH_DASH_ARGS" or something like that. Future readers will easily know what that means.